test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ExportTests.cs

```diff
@@ -41,10 +40,12 @@ public async Task GivenExportIsEnabled_WhenRequestingExportForResourceWithCorrec
             Assert.Equal(HttpStatusCode.MethodNotAllowed, response.StatusCode);
         }
 
-        [Fact]
-        public async Task GivenExportIsEnabled_WhenRequestingExportWithCorrectHeaders_ThenServerShouldReturnAcceptedAndNonEmptyContentLocationHeader()
+        [Theory]
+        [InlineData("$export")]
+        [InlineData("Patient/$export")]
```

Comment:
Would it be useful to add a test for the endpoint `Observation/$export` and validate that it returns an unsuccessful response?